src/App.js

```diff
@@ -85,13 +85,17 @@ class App extends Component {
         <div id="controllers">
           <div id="total-display" className="row">
             <div className="col-8" style={{padding: 0}}>
-              <button style={{float: "left"}} onClick={() => {this.setState({mod: this.state.mod - 1})}}>
+              <button className="mod-minus" onClick={() => {
+                  this.setState({mod: this.state.mod - 1})
+                }}>
                 -
               </button>
-              {this.renderDesc()}
-              <button style={{float: "right"}} onClick={() => {this.setState({mod: this.state.mod + 1})}}>
+              <button className="mod-plus" onClick={() => {
+                  this.setState({mod: this.state.mod + 1})
+                }}>
                 +
               </button>
+              {this.renderDesc()}
```

Comment:
Oui oui, quest ce c'est, this renderDesc? I do not comprehend the ideas utilized in this passage

Reply:
Oui oui, I second this.